frontend/templates/layouts/rejected.html

```diff
@@ -0,0 +1,39 @@
+<html>
+    <link <link href="https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap" rel="stylesheet" />
+    <style>
+        body {
+            padding: 50px;
+            font-family: "Akzidenz Roman", Helvetica, Arial, sans-serif;
+            font-size: 14px;
+            line-height: 1.5;
+        }
+    
+        .rejected {
+            color: darkred;
+        }
+    </style>
+    </head>
+    <body>
+        <p>As-salamu alaykum {{modifiedBy}},</p>
+        <br />
+        <p>
+            Thank you for submitting a corrections request for the following Hadith.
+        </p>
+        <p>
+            <span><b>Hadith: </b>{{attr | safe}}</span><br />
+            <span><b>Modifications: </b>{{modifiedText | safe}}</span>
+        </p>
+        <br />
+        <p>
+            We would like to inform you that your request has been <span class='rejected'>rejected</span> by our moderators for the following reason.
+            <p><b>Reason:</b> {{message}}</p>
+        </p>
+
+        <br />
+        <p>We sincierely appreciate your time for supporting our mission.</p>
+        <p>
+            <span>Regards,</span> <br />
+            <span><b>Team Sunnah.com</b></span>
+        </p>
+    </body>
+</html>
```

Comment:
- can you add a new line to the end of the file
- can you move the template to frontend/templates/email/rejected.html instead of layouts/rejected?
- can you squash the commits

Reply:
Ready for review